Sources/TuistGenerator/Generator/DerivedFileGenerator.swift

```diff
@@ -57,8 +64,7 @@ final class DerivedFileGenerator: DerivedFileGenerating {
     func generateInfoPlists(project: Project, sourceRootPath: AbsolutePath) throws -> Set<AbsolutePath> {
         let infoPlistsPath = DerivedFileGenerator.infoPlistsPath(sourceRootPath: sourceRootPath)
         let targetsWithGeneratableInfoPlists = project.targets.filter {
-            guard let infoPlist = $0.infoPlist else { return false }
-            guard case InfoPlist.dictionary = infoPlist else { return false }
```

Comment:
ouch, this regresses https://github.com/tuist/tuist/issues/449

```swift
switch $0.infoPlist {
   case nil, .file:
      return false
   default:
      return true
}
```

Reply:
Good catch :+1:. The tests that covered this was actually failing. Glad that we have the project covered with tests.